src/bundle/eZ/RichText/Renderer.php

```diff
@@ -162,6 +164,24 @@ function (Repository $repository) use ($contentId) {
             );
 
             $isDenied = true;
+        } catch (AuthenticationCredentialsNotFoundException $e) {
```

Comment:
it looks like this try/catch construction has grown a lot, it could be good to split it a bit into private functions